packages/serverless/src/apollo/app/user.ts

```diff
@@ -179,10 +179,16 @@ export const typeDefsUser = gql`
     publicKey: String
   }
 
+  type UserPaginationOutput {
+    data: [User]!
+    totalSize: Int!
+    offset: Int!
+  }
+
   extend type Query {
     userSignInData: SignInResponse
-    findUser(query: FindUser!, pagination: PaginationInput): [User]!
-    searchUser(query: FindUser!, pagination: PaginationInput): [User]!
+    findUser(query: JSON, pagination: PaginationInput): UserPaginationOutput!
```

Comment:
How can we query users without manually constructing queries from strings? The FindUser method works well for search since the query is generated for it, but not for `find` one. We plan to replace query json across all endpoints soon. Take a look at apollo/app/types/search.ts — I'm considering adopting  it or a similar approach to replace the json

Reply:
Okay can you put it to TODO: